src/commands/appdistribution-testers-list.ts

```diff
@@ -47,7 +47,7 @@ function printTestersTable(testers: Tester[]): void {
 
   for (const tester of testers) {
     const name = tester.name.split("/").pop();
-    const groups = tester.groups
+    const groups = (tester.groups || [])
```

Comment:
Nit: Prefer ?? when adding a fallback for undefined cases
```suggestion
    const groups = (tester.groups ?? [])
```